express_relay/sdk/js/src/index.ts

```diff
@@ -218,33 +268,66 @@ export class Client {
     this.websocketOpportunityCallback = callback;
   }
 
+  public setBidStatusHandler(
+    callback: (statusUpdate: BidStatusUpdate) => Promise<void>
+  ) {
+    this.websocketBidStatusCallback = callback;
+  }
+
   /**
    * Subscribes to the specified chains
    *
    * The opportunity handler will be called for opportunities on the specified chains
    * If the opportunity handler is not set, an error will be thrown
    * @param chains
    */
-  async subscribeChains(chains: string[]) {
+  async subscribeChains(chains: string[]): Promise<void> {
     if (this.websocketOpportunityCallback === undefined) {
-      throw new Error("Opportunity handler not set");
+      throw new ClientError("Opportunity handler not set");
     }
-    return this.sendWebsocketMessage({
+    await this.sendWebsocketMessage({
       method: "subscribe",
       params: {
         chain_ids: chains,
       },
     });
   }
 
+  async submitOpportunityBidViaWebsocket(bid: OpportunityBid): Promise<BidId> {
```

Comment:
i think there's probably better clearer naming here, but that probably depends on us developing clearer terminology for these different parts of the server. rn the terms OpportunityBid vs Bid are not very clear at first inspection, maybe something like submitLiquidationAdapterBid vs. submitBid?

Reply:
I agree that our terminology is a bit confusing but not sure about your suggestion. the term `adapter` has never been exposed in the api so I think it makes it more confusing here. I guess I have made a decision early on the summarise the LiquidationOpportunity term by dropping the liquidation part. LiquidationOpportunityBid is more verbose but also a bit long. I'm open to suggestions but terminology changes should be consistent across different docs and services.

Reply:
I think this depends on what we call the LiquidationAdapter server to differentiate it from the auction server. maybe we ought to refine that naming distinction, and that will make it clearer: for example, naming it "Liquidation Forum"?

Reply:
exposing the `viaWebsocket` to users seems like it's exposing an implementation detail of how this works. There's a functional difference between this and the post method below though, which is that you get updates on the bid if you use websocket. 

I think a better way to expose this functionality would be to have one method like `submitOpportunityBid` that has a default parameter like `subscribeToUpdates = true`, and then switches between websocket and post depending on that.